Modules/IO/NIFTI/test/itkNiftiLargeImageRegionReadTest.cxx

```diff
@@ -0,0 +1,68 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         https://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+
+#include "itkImageFileReader.h"
+#include "itkImageFileWriter.h"
+#include "itkTestingMacros.h"
+
+
+int
+itkNiftiLargeImageRegionReadTest(int argc, char * argv[])
+{
+
+  if (argc != 2)
+  {
+    std::cerr << "Missing arguments" << std::endl;
+    std::cerr << "Usage: " << itkNameOfTestExecutableMacro(argv) << " outputFileName" << std::endl;
+    return EXIT_FAILURE;
+  }
+
+
+  constexpr int Dimension = 3;
+  using PixelType = unsigned short;
+
+  using ImageType = itk::Image<PixelType, Dimension>;
+
+  // Create a large image
+  ImageType::SizeType   size = { { 1034, 1034, 1020 } };
+  ImageType::RegionType region;
+  region.SetSize(size);
+
+  auto image = ImageType::New();
+  image->SetRegions(region);
+  image->Allocate(true);
+
+  const std::string fname{ argv[1] };
+  itk::WriteImage(image, fname);
+
+
+  using ReaderType = itk::ImageFileReader<ImageType>;
+  auto reader = ReaderType::New();
+
+  reader->SetFileName(fname);
+
+  // Ensure that a sufficiently large subregion of an abritrary pixel type image can be read
```

Comment:
Small typo here "abritrary" (no show-stopper to me). Instead of fixing the typo, I'd rather just shorten the sentence to:

>     // Ensure that a sufficiently large subregion of an image can be read.

But that's just my preference, no issue.

Reply:
Re-worded. The word is no longer present.